components/utils/stripe.ts

```diff
@@ -0,0 +1,5 @@
+import Stripe from 'stripe'
+
+export const stripe = new Stripe(process.env.STRIPE_SECRET_KEY ?? "", {
+  apiVersion: '2020-08-27',
```

Comment:
`apiVersion` required for TS [according to the docs](https://github.com/stripe/stripe-node#usage-with-typescript)